src/test/java/com/ericsson/ei/subscription/SubscriptionValidatorTest.java

```diff
@@ -142,8 +131,7 @@ public void validateNotificationMessageValidMessageTest() throws Exception {
             invokeMethod(SubscriptionValidator.class, "validateNotificationMessageKeyValues",
                     subscription.getNotificationMessageKeyValues(), "");
         } catch (SubscriptionValidationException e) {
-            assertTrue(e.getMessage(), false);
-            return;
+            fail(e.getMessage());
```

Comment:
Here and elsewhere, why bother to catch?

Reply:
Maybe not needed since any exception will fail to test, but updating old tests is not prioritized.

Reply:
Isn't it easier to remove the catch then to add the fail?